src/protocols/strategies/convex.strategy.ts

```diff
@@ -269,24 +268,34 @@ export async function resolveCurvePoolTokenPrice(chain: Chain, token: Token): Pr
 }
 
 // TODO: this function is a bit weird, we can't assume to ever have a 'Harvest'
-async function retrieveBribesProcessorData(chain: Chain, vault: VaultDefinition): Promise<CachedValueSource[]> {
+async function retrieveHarvestForwarderData(chain: Chain, vault: VaultDefinition): Promise<CachedValueSource[]> {
   const sdk = await chain.getSdk();
-  const bribeProcessor = BribesProcessor__factory.connect('0xbed8f323456578981952e33bbfbe80d23289246b', sdk.provider);
+  const harvestForwarder = HarvestDistributor__factory.connect(
+    '0xA84B663837D94ec41B0f99903f37e1d69af9Ed3E',
+    sdk.provider,
+  );
 
-  const treeDistributionFilter = bribeProcessor.filters.TreeDistribution();
+  const treeDistributionFilter = harvestForwarder.filters.TreeDistribution();
 
   const endBlock = await sdk.provider.getBlockNumber();
   const startBlock = Math.floor(endBlock - (21 * ONE_DAY_SECONDS) / 13);
   const allTreeDistributions = await chunkQueryFilter<
-    BribesProcessor,
+    HarvestDistributor,
     TreeDistributionEventFilter,
     TreeDistributionEvent
-  >(bribeProcessor, treeDistributionFilter, startBlock, endBlock);
+  >(harvestForwarder, treeDistributionFilter, startBlock, endBlock);
 
-  const { harvests, distributions } = await parseHarvestEvents([], allTreeDistributions);
+  const distributions = allTreeDistributions
+    .filter((d) => d.args.beneficiary === TOKENS.BVECVX)
```

Comment:
There will be bvecvxLP beneficiery events as well at a minimum, but the contract is designed to be general purpose, meaning any future products can utilize it to distribute an event without having to be in a vault or conform to any registries